app/src/main/res/xml/shortcuts.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+    <shortcut
+        android:enabled="true"
+        android:icon="@drawable/ic_playlist_shortcut"
+        android:shortcutId="queue"
+        android:shortcutShortLabel="@string/queue_label">
+        <intent
+            android:action="android.intent.action.VIEW"
+            android:targetClass="de.danoeh.antennapod.activity.MainActivity"
+            android:targetPackage="${applicationId}">
+            <extra
+                android:name="fragment_tag"
+                android:value="QueueFragment" />
+            <extra
+                android:name="nav_type"
+                android:value="0" />
+        </intent>
+    </shortcut>
+
+    <shortcut
+        android:enabled="true"
+        android:icon="@drawable/ic_feed_shortcut"
+        android:shortcutId="episodes"
+        android:shortcutShortLabel="@string/episodes_label">
+        <intent
+            android:action="android.intent.action.VIEW"
+            android:targetClass="de.danoeh.antennapod.activity.MainActivity"
+            android:targetPackage="${applicationId}">
+            <extra
+                android:name="fragment_tag"
+                android:value="EpisodesFragment" />
+            <extra
+                android:name="nav_type"
+                android:value="0" />
+        </intent>
+    </shortcut>
+
+    <shortcut
+        android:enabled="true"
+        android:icon="@drawable/ic_folder_shortcut"
+        android:shortcutId="subscriptions"
+        android:shortcutShortLabel="@string/subscriptions_label">
+        <intent
+            android:action="android.intent.action.VIEW"
+            android:targetClass="de.danoeh.antennapod.activity.MainActivity"
+            android:targetPackage="${applicationId}">
+            <extra
+                android:name="fragment_tag"
+                android:value="SubscriptionFragment" />
+            <extra
+                android:name="nav_type"
+                android:value="0" />
```

Comment:
This argument is now unused. Could you please remove it from all intents?

Reply:
Thank you for pointing this out. This argument has certainly been removed in #3943 . Fixed in 1ac2ef7 .